example-app/index.js

```diff
@@ -43,26 +44,43 @@ app.use(expressSession({
   resave: false,
   saveUninitialized: false,
   secret: 'REPLACE THIS SECRET WITH YOUR OWN!', // secret to encrypt session cookies
-  cookie: {
-    sameSite: 'lax' // needed because otherwise the different port means the cookie won't be forwarded to OPA
-  }
 }));
 
 // endpoint to serve the single page app
 app.get('/', (_req, res) => res.sendFile(path.join(__dirname, 'index.html')));
 app.get('/ai-request.js', (_req, res) => res.sendFile(path.join(__dirname, 'ai-request.js')));
 
-// endpoint to check if the request contains an authenticated session [Ref-2.1] [Ref-2.2]
```

Comment:
we can go either way with these references, either put a comment on whats could be implemented to show developers which component does what.  or just delete from the reference diagram if its no longer useful

Reply:
I reworked the diagram and added in new references.